source/kraken/apply_disruption.cpp

```diff
@@ -186,11 +186,20 @@ static nt::Route* get_or_create_route(const nt::disruption::Impact& impact, nt::
     nt::Network* network = pt_data.get_or_create_network("network:additional_service", "additional service");
     nt::CommercialMode* comm_mode =
         pt_data.get_or_create_commercial_mode("commercial_mode:additional_service", "additional service");
+
+    // We get the first and last stop_area to create route and line
+    const auto& st_depart = impact.aux_info.stop_times.front();
+    const auto& sa_depart = st_depart.stop_time.stop_point->stop_area;
+    const auto& st_arrival = impact.aux_info.stop_times.back();
+    const auto& sa_arrival = st_arrival.stop_time.stop_point->stop_area;
+    std::string line_uri = "line:" + sa_depart->uri + "_" + sa_arrival->uri;
+    std::string line_name = "line:" + sa_depart->name + "_" + sa_arrival->name;
```

Comment:
I think we should do something like this:
```suggestion
    std::string line_name = sa_depart->name + " - " + sa_arrival->name;
```
It would look more like lines built the same way: https://api.navitia.io/v1/coverage/sncf/networks/network%3Asncf/lines